apps/openmw/mwrender/groundcover.cpp

```diff
@@ -193,21 +182,22 @@ namespace MWRender
         {
             for (int cellY = startCell.y(); cellY < startCell.y() + size; ++cellY)
             {
-                const ESM::Cell* cell = mGroundcoverStore.get<ESM::Cell>().searchStatic(cellX, cellY);
-                if (!cell) continue;
+                ESM::Cell cell;
+                mGroundcoverStore.initCell(cell, cellX, cellY);
```

Comment:
Is there a particular reason to use a mutable reference argument here, instead of returning by reference?

Reply:
I can not just return reference to local variable - it will be destroyed outside the `initCell` scope. Do you know a better way to avoid this issue?